### ISSUE TITLE
IntelliJ IDEA 14.1.1 push to build server issue

### ISSUE THREAD

**Issue by rhg1968** [codenameone/CodenameOne#1448]
I am just coming back to re-evaluate Codename one and I am running into some issues. I can create a new project and debug it, but when I choose to send it to the build server I am getting the error message that it can't find the build.xml file. That file is in the project and I am showing no updates for the code name one plugin, so I'm not sure what the issue is. Any insights would be greatly appreciated.

There has been an ongoing discussion about this topic in the Codenameone forums located at the url below and titled: IntelliJ build issue

https://groups.google.com/forum/embed/?place=forum%2Fcodenameone-discussions&showsearch=true&showpopout=true&hl=en&parenturl=http%3A%2F%2Fcodenameone.com%2Fdiscussion-forum.html#!topic/codenameone-discussions/bsRog6Fp7zw
![cnoneerror](https://cloud.githubusercontent.com/assets/2014195/7164423/18a4a4d4-e36e-11e4-83ca-8888c10fdfac.jpg)

**Comment by rhg1968** [comment]
I downloaded the 3.0.0 version of the plugin and it still can't deploy to the build server and now in addition I get the same can't find build.xml file when trying to run it in the simulator

**Comment by rhg1968** [comment]
An interesting side note. I installed a new version of IntelliJ IDEA on a different machine and got the same results. When I have the project loaded and go into the File Settings menu option, there is an other settings section in the pop up window. Under it are two entries for Codename One.  One has settings for IOS, Android, and BlackBerry. The other one has settings for IOS, Android, BlackBerry, Windows Phone, and J2ME. Not sure if this is the way it should be.

**Comment by rhg1968** [comment]
I am going to add just a little more info. Just for the heck of it I installed Netbeans and downloaded the plugin. It said it was unsigned or something with the certificate, can't really remember, and it has a much lower version number than the intelliJ one. But I was able to push a build to the server and install it on my device. I definitely prefer working in the IntelliJ editor, so I would definitely like to find out if you have any ideas on what the issue might be. Once again I definitely appreciate your help.
